azure-pipelines/pull-request-validation/build-consumers.yml

```diff
@@ -210,10 +223,9 @@ stages:
           cwd: $(Build.SourcesDirectory)/broker-java-root
           tasks: LinuxBroker:clean LinuxBroker:assemble --build-cache --info
           publishJUnitResults: false
-          jdkArchitecture: x64
-          jdkVersionOption: "1.11"
+          jdkArchitecture: x86
+          sqAnalysisBreakBuildIfQualityGateFailed: false
       - task: Bash@3
-        retryCountOnTaskFailure: 3
```

Comment:
Any reason for skipping the retry?

Reply:
Seemed out of place. it's only place in our pipelines where we use this property from what i understand. I added it back anyway